src/awsexplorer/activation.ts

```diff
@@ -209,3 +219,8 @@ function updateAwsExplorerWhenAwsContextCredentialsChange(
         })
     )
 }
+
+async function copyArnToClipboard(arn: string) {
+    await vscode.env.clipboard.writeText(arn)
+    vscode.window.setStatusBarMessage(localize('AWS.explorerNode.copiedArn', 'Copied ARN to clipboard'), 2000)
```

Comment:
In the case where the arn cannot be found, I think we should show an error message and also not update the clipboard.